lib/components/T3MediaFile/type/Vimeo.vue

```diff
@@ -1,11 +1,10 @@
-<template functional>
+<template>
   <iframe
-    v-bind="data.attrs"
-    :class="[data.class, data.staticClass]"
-    :src="props.file.publicUrl"
-    :width="props.file.properties.dimensions.width || 640"
-    :height="props.file.properties.dimensions.height || 360"
-    :title="props.file.properties.title || false"
+    v-bind="$attrs"
```

Comment:
is there a need to use v-bind="$attrs" since we don't use inheritAttrs:false here ?